ci/jenkins/README.md

```diff
@@ -49,3 +59,9 @@ Run the command to apply these jobs.
 ```bash
 jenkins-jobs update -r ci/jenkins/jobs
 ```
+
+### Jenkins job updater
+For the convenience of letting Jenkins jobs keep pace with code in repo, there
```

Comment:
I suggest replacing "For the convenience of letting Jenkins jobs keep pace with code in repo," with "To follow GitOps best practices,"

Reply:
I agree. Updated